includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -3129,6 +3130,136 @@ static function ( $attribute_name ) {
+				);
+
+			case AMP_Tag_And_Attribute_Sanitizer::INCORRECT_MIN_NUM_CHILD_TAGS:
+				return sprintf(
+					/* translators: %1$s is the node name, %2$s is required child count */
+					_n(
+						'Tag <code>%1$s</code> must have a minimum of %2$d child tag',
+						'Tag <code>%1$s</code> must have a minimum of %2$d child tags',
+						(int) $validation_error['required_min_child_count'],
+						'amp'
+					),
+					esc_html( $validation_error['node_name'] ),
+					esc_html( $validation_error['required_min_child_count'] )
+				);
+
+			case AMP_Tag_And_Attribute_Sanitizer::WRONG_PARENT_TAG:
+				return sprintf(
+					/* translators: %1$s is the node name, %2$s is parent name */
+					__( 'The parent tag of tag <code>%1$s</code> cannot be <code>%2$s</code>', 'amp' ),
+					esc_html( $validation_error['node_name'] ),
+					esc_html( $validation_error['parent_name'] )
+				);
+
+			case AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_TAG_ANCESTOR:
+			case AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_DESCENDANT_TAG:
+				return sprintf(
+					/* translators: %1$s is the node name, %2$s is the disallowed ancestor tag name */
+					__( 'The tag <code>%1$s</code> may not appear as a descendant of tag <code>%2$s</code>', 'amp' ),
+					esc_html( $validation_error['node_name'] ),
+					esc_html( $validation_error['disallowed_ancestor'] )
+				);
+
+			case AMP_Tag_And_Attribute_Sanitizer::MANDATORY_TAG_ANCESTOR:
+				return sprintf(
+					/* translators: %1$s is the node name, %2$s is the required ancestor tag name */
+					__( 'The tag <code>%1$s</code> may only appear as a descendant of tag <code>%2$s</code>', 'amp' ),
+					esc_html( $validation_error['node_name'] ),
+					esc_html( $validation_error['required_ancestor_name'] )
+				);
+
+			case AMP_Tag_And_Attribute_Sanitizer::INVALID_ATTR_VALUE:
+			case AMP_Tag_And_Attribute_Sanitizer::INVALID_ATTR_VALUE_CASEI:
+			case AMP_Tag_And_Attribute_Sanitizer::INVALID_ATTR_VALUE_REGEX:
+			case AMP_Tag_And_Attribute_Sanitizer::INVALID_ATTR_VALUE_REGEX_CASEI:
+			case AMP_Tag_And_Attribute_Sanitizer::INVALID_BLACKLISTED_VALUE_REGEX:
+				return sprintf(
+					/* translators: %1$s is the attribute name, %2$s is the invalid attribute value */
+					__( 'The attribute <code>%1$s</code >is set to the invalid value <code>%2$s</code>', 'amp' ),
+					esc_html( $validation_error['node_name'] ),
+					esc_html( $validation_error['element_attributes'][ $validation_error['node_name'] ] )
+				);
+
+			case AMP_Tag_And_Attribute_Sanitizer::INVALID_URL_PROTOCOL:
+				$parsed_url       = wp_parse_url( $validation_error['element_attributes'][ $validation_error['node_name'] ] );
+				$invalid_protocol = isset( $parsed_url['scheme'] ) ? $parsed_url['scheme'] : '';
+
+				return sprintf(
+					/* translators: %1$s is the invalid protocol, %2$s is attribute name */
+					__( 'Invalid URL protocol <code>%1$s:</code> for attribute <code>%2$s</code>', 'amp' ),
+					$invalid_protocol,
+					esc_html( $validation_error['node_name'] )
```

Comment:
In addition to moving the `code` markup, the `:` should be moved below:

```suggestion
					__( 'Invalid URL protocol %1$s for attribute %2$s</code>', 'amp' ),
					'<code>' . esc_html( $invalid_protocol ) . ':</code>',
					'<code>' . esc_html( $validation_error['node_name'] ) . '</code>'
```

Or rather the `:` should be made part of the `$invalid_protocol` above.